### DISCUSSION TITLE
How do you use this?

### DISCUSSION THREAD

**Discussion by jerifozzle** [swingmx/swingmusic#213] [Q&A]
There's no .exe file and the app folder does nothing

**Comment by cwilvx** [comment]
Hi @jerifozzle 

You can get a windows executable from the releases page under the assets section.

https://github.com/swingmx/swingmusic/releases

**Reply by jerifozzle** [reply]
![image](https://github.com/swingmx/swingmusic/assets/170806748/05caa3e3-44c5-47f2-8969-956fcf8ca132)

**Reply by cwilvx** [reply]
Hi @jerifozzle 

Click the localhost url (http://localhost:1970) on your cmd to access the user interface.

You'll be prompted to select where you have stored your music.

**Reply by jerifozzle** [reply]
That didn't work

**Comment by michilyy** [comment]
Hello @jerifozzle ,
does this problem still exists with the newest version?
